.firebase/hosting.YnVpbGQ.cache

```diff
@@ -1,25 +1,26 @@
-asset-manifest.json,1594568444793,56b67a35f2d030643d96b90ca0b96f1e9fc322f0d02cd84f50fa736e1aa3ec0a
-index.html,1594568444773,8fab7db0238ce9a624b03a32977352034ab76efa046a51dceded305047372b58
-manifest.json,1594568398583,341d52628782f8ac9290bbfc43298afccb47b7cbfcee146ae30cf0f46bc30900
-precache-manifest.210549c1b68b1d042c02cf9adf2fe26d.js,1594568444773,2cabda91da7f640c2425be9a077d65aaea51ea0de5727e57fdeeca2c60865796
-service-worker.js,1594568444793,b5e1c46660e0c6ad6c3574277866b773b93f9bff9fd605b5060c85e6219b6339
-static/css/main.eeebc31b.chunk.css,1594568444772,a0d816ee9031690351564966ea6951a75da0390b1bad6a185146f04e594c028f
-static/css/main.eeebc31b.chunk.css.map,1594568444772,46562146e7ebeadf7c03dad12cb66b4a71d14f638b23bd2741b7687855c16f80
-logo512.png,1594568398582,191fc21360b4ccfb1cda11a1efb97f489ed22672ca83f4064316802bbfdd750e
-logo192.png,1594568398581,7d1ce084201cab0c66b5e2d536713951160a5c541d09d745e9f63766a27d4661
-favicon.ico,1594568398579,79fdc3a9e8e9ddb3da4ecbcfd4eaaa348e60e4033d8da5671d75be26e9b69a2f
-static/js/2.aa963a28.chunk.js.LICENSE.txt,1594568444772,36f1c5e5835f6df8a81802dc284acc8ad16dd4a667e5fc348d3d684a459dab86
-static/js/main.2d564485.chunk.js,1594568444771,a9873ed4d2e709cf3ca5056c1e1b0a2addd1c6ab299d3ee1b4259c659c86f029
-static/js/runtime-main.6f3bb257.js,1594568444771,c0fd40f0c071d4374cd21a0f6b3c8016f351d99700ec70fd9cf1d41604d221e3
-static/js/runtime-main.6f3bb257.js.map,1594568444772,7d057fcc4f20c910e396c3fbe65f75de8a6145a94064cfde487c8366c4d4079e
-static/media/logo.dcad33a1.png,1594568444772,7d1ce084201cab0c66b5e2d536713951160a5c541d09d745e9f63766a27d4661
-robots.txt,1594568398584,391d14b3c2f8c9143a27a28c7399585142228d4d1bdbe2c87ac946de411fa9a2
-static/media/smu_logo.78c2c82e.png,1594568444772,c8bc99431f22bfe5f64965f571fe080120aec92417fe08e3c8559d3307b84b46
-static/media/productCurvyLines.3a1df6c1.png,1594568444765,0852b5dd194f4975f615f0479f9faaa585c34e212f99ec0f3074d32989cf2c6b
-static/js/main.2d564485.chunk.js.map,1594568444772,54febf6b456d875be2ff775ba4a64e7b1bfce6e4befe3e3ad1cc37b6302b34ab
-static/media/nus_logo.7665f004.png,1594568444772,de6108a27c4587630d0093c6d7c797caa05e264197c40316cfeb60ad34fbb15b
-static/css/2.9edc93b8.chunk.css,1594568444772,d7eb40748c9a1798d50e96ad6dd1e1c2cf38ae59ff55d32b752291fb26156f6d
-static/media/ntu_logo.72c0aa58.png,1594568444771,50c8439bf4557d50d46b320445a63b6540fa0439d0b74ab682f11f2bb350042b
-static/js/2.aa963a28.chunk.js,1594568444772,10177f82734c6dda0f938c0a0031d4121f7277f2b8073af1be351eb79ebb708d
-static/css/2.9edc93b8.chunk.css.map,1594568444772,4444261aafb4439aec6ffa94ca02d0933b8e0b4c8d1d24cc04379e0c2075d1e6
-static/js/2.aa963a28.chunk.js.map,1594568444794,d40a45e19659206f8036b0bdc4ab3a5a5d66e30ffb484d7fa73672eb63f1a15b
+asset-manifest.json,1594917935146,b6357f9825e9a930f128039d9026f5248a03b8d2fdf99b2d66b62122e4e0a9a4
```

Comment:
nit: Should this be added in gitignore? 😮

Reply:
Oh right, will gitignore it in my next PR, thanks!